assets/stylesheets/shared/_colors.scss

```diff
@@ -1,65 +1,73 @@
+/** @format */
+
+// The Muriel Color Palette
+@import '../../../node_modules/color-studio/dist/colors';
```

Comment:
I think the sass-loader can do something like `~color-studio/dist/colors`. Let's also use an explicit extension for the file we load.

Reply:
That stylesheet is referenced in `package.json`, so `color-studio` should also work in that case. 🙂

Reply:
@flootr @porada As per documentation, both methods should work, yes. In practice though, they did error out for me though when testing them locally previously. 🤔Feel free to test it out on this branch, good chance that I'm probably missing something. :)

Reply:
The reason this doesn't work is that our `_colors.scss` is not processed with Webpack so we can't use this resolve mechanism from `sass-loader` (yet). Sorry, I didn't see this when I first looked at it.